migtests/scripts/run-test.sh

```diff
@@ -169,6 +169,10 @@ main() {
 	run_ysql ${TARGET_DB_NAME} "\di"
 	run_ysql ${TARGET_DB_NAME} "\dft" 
 
+	# sleep of 1100ms; greater than hearbeat interval either of 1000ms or 500ms(depends on yb version)
+	# sleep > max(500, 1000) * n 
```

Comment:
Can you put the conclusion of our description either in the PR or in the comment here? 
Also, ` * n` ?

Reply:
raised a separate PR.
https://github.com/yugabyte/yb-voyager/pull/2629

will close this PR after we have fully concluded.